diagnostic.md

```diff
@@ -71,7 +71,7 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 If we are in the `project` directory and use `cd planning`, is a relative or absolute path being referenced? How do you know?
 
  <!-- Answer Starts Here -->
-
+i dunno
```

Comment:
Relative path, planning is referenced with respect to the current working directory, project.